server/src/main/java/fr/uga/l3miage/tp2/exo1/models/ClubSportif.java

```diff
@@ -0,0 +1,24 @@
+package fr.uga.l3miage.tp2.exo1.models;
+
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "ClubSportif")
```

Comment:
```suggestion
@Table(name = "club_sportif")
```
même si il n'est pas utilise ici, on ne pet pas de `camelCase` dans le nom des attributs en base